Controls/Mail/Mailman.c

```diff
@@ -0,0 +1,25 @@
+#include "Mail.h"
+
+/**
+ * The mailman will put mail in the mailbox.
+ */
+
+
+/**
+ * @brief Puts a letter in the mailbox (declared in Tasks.h). The letter must be unique every time
+ * this function is called (suggestion: include a random number!). Print "Finished!" at the end.
+ */
+void depositLetter(void) {
+	OS_ERR err; // Make sure to check for errors and print the error code if not OS_ERR_NONE
+}
+
+/**
+ * @brief Calls depositLetter. Once the letter is deposited, signal the semaphore (effectively putting up the flag).
```

Comment:
I think it makes sense to move the semaphore signaling into depositLetter, that way the task only has to call it and then delay. But either way should work.